code/datums/effects/bleeding.dm

```diff
@@ -70,16 +70,17 @@
 		if(affected_mob.reagents) // Annoying QC check
 			if(affected_mob.reagents.get_reagent_amount("thwei"))
 				blood_loss -= THWEI_BLOOD_REDUCTION
-
-			if(affected_mob.bodytemperature <= BODYTEMP_CRYO_LIQUID_THRESHOLD && (affected_mob.reagents.get_reagent_amount("cryoxadone") || affected_mob.reagents.get_reagent_amount("clonexadone")))
+			if(affected_mob.bodytemperature <= (BODYTEMP_CRYO_LIQUID_THRESHOLD) && (affected_mob.reagents.get_reagent_amount("cryoxadone") || affected_mob.reagents.get_reagent_amount("clonexadone")) )
 				var/obj/structure/machinery/cryo_cell/cryo = affected_mob.loc
 				if(istype(cryo) && cryo.on && cryo.operable())
 					blood_loss -= CRYO_BLOOD_REDUCTION
 
 			var/mob/living/carbon/human/affected_human = affected_mob
 			if(istype(affected_human))
+				///Changed CHEM_EFFECT_NO_BLEEDING effect to also cure bleeding via blood_loss reduction.
```

Comment:
autodoc only variables, this *probably* works(in some weird way), but just do a normal comment. Also, an extremely minor nitpick, but write comments as being true, not what happened to it
```suggestion
				//CHEM_EFFECT_NO_BLEEDING effect cure bleeding via blood_loss reduction.
```